Extensions/FixMyPosts.js

```diff
@@ -0,0 +1,64 @@
+//* TITLE FixMyPosts **//
+//* VERSION 999.999.999 **//
+//* DESCRIPTION Fixes the reblog format of posts to reflect the blockquote-style **//
+//* DEVELOPER THETIMEISNEVERIGHT **//
```

Comment:
It's all-caps in the original extension. I don't think there's a good reason to change it at this point?